typespec-tests/src/main/java/com/_specs_/azure/core/lro/rpc/RpcClient.java

```diff
@@ -68,7 +68,7 @@ public final class RpcClient {
      * }
      * }</pre>
      * 
-     * @param generationOptions Options for the generation.
```

Comment:
curious why would the description change to default?

Reply:
Good question.

The "after" is due to this https://github.com/Azure/cadl-ranch/pull/603/files#r1660519786

The "before" seems from here
https://github.com/weidongxu-microsoft/cadl-ranch/blob/d8ca94bef39fbdfb188ca4288ec77858e2584d33/packages/cadl-ranch-specs/http/azure/core/lro/rpc/main.tsp#L29-L33
But I am not sure about the process that the doc on model end up to the doc on the parameter...

Reply:
I see. I guess it's because we get description from body type. https://github.com/Azure/autorest.java/blob/main/typespec-extension/src/code-model-builder.ts#L817

I will pay attention when migrating to `sdkpackage`, to use `bodyParam.description`, instead of `bodyParam.type.description`

Reply:
Ah, yes. On implicitly body, the model itself is treated as body.